integrationTests/sync/basicSync_test.go

```diff
@@ -0,0 +1,104 @@
+package sync
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/ElrondNetwork/elrond-go/integrationTests"
+	"github.com/ElrondNetwork/elrond-go/sharding"
+	"github.com/stretchr/testify/assert"
+)
+
+var stepDelay = time.Second
+var delayP2pBootstrap = time.Second * 2
+
+func TestSyncWorksInShard_EmptyBlocksNoForks(t *testing.T) {
+	if testing.Short() {
+		t.Skip("this is not a short test")
+	}
+
+	maxShards := uint32(1)
+	shardId := uint32(0)
+	numNodesPerShard := 6
+
+	advertiser := integrationTests.CreateMessengerWithKadDht(context.Background(), "")
+	_ = advertiser.Bootstrap()
+	advertiserAddr := integrationTests.GetConnectableAddress(advertiser)
+
+	nodes := make([]*integrationTests.TestProcessorNode, numNodesPerShard+1)
+	for i := 0; i < numNodesPerShard; i++ {
+		nodes[i] = integrationTests.NewTestSyncNode(
+			maxShards,
+			shardId,
+			shardId,
+			advertiserAddr,
+		)
+	}
+
+	metachainNode := integrationTests.NewTestSyncNode(
+		maxShards,
+		sharding.MetachainShardId,
+		shardId,
+		advertiserAddr,
+	)
+	nodes[numNodesPerShard] = metachainNode
+
+	idxProposerShard0 := 0
+	idxProposerMeta := numNodesPerShard
```

Comment:
move this line above so you can use idxProposerMeta on line 46

Reply:
done